### PR TITLE
Feat/add variable enable nat gateway

### PR DESCRIPTION
**Pull request by lassebenni**

### CODE REVIEW

**Comment by lassebenni** [comment]
FYI @svanharmelen, @shoekstra  :

As per request of @borisroman adding variable to be able to prevent creation of NAT Gateways when it is not preferred.

File: variables.tf [hunk 1]
```diff
@@ -8,6 +8,12 @@ variable "cidr_block" {
   description = "The CIDR block for the VPC"
 }
 
+variable "enable_nat_gateway" {
+  type        = bool
+  default     = true
+  description = "Should be true to provision NAT Gateways for each private subnet"
```
Comment (shoekstra) [inline variables.tf:14]: Hi @lassebenni! Very minor suggestion, we prefer to say "Set to ...":

```suggestion
  description = "Set to true to provision a NAT Gateway for each private subnet"
```

**Review by shoekstra — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (lassebenni) [reply variables.tf:14]: @shoekstra Definitely a good practice, thanks!

**Review by svanharmelen — DISMISSED** [review]
Looks good @lassebenni 👍🏻

**Review by svanharmelen — APPROVED** [review]

**Review by shoekstra — APPROVED** [review]

**Pull request merged**